recipes/tuiview/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.1.8" %}
+
+package:
+    name: tuiview
+    version: {{ version }}
+
+source:
+    fn: TuiView-{{ version }}.tar.gz  # [unix]
+    fn: TuiView-{{ version }}.zip  # [win]
+    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-1.1.8.tar.gz  # [unix]
+    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-1.1.8.zip  # [win]
```

Comment:
bat files are fine if they're necessary, but they're usually not.  If your code runs a particular Python file or function, you can use entry points instead: http://conda.pydata.org/docs/building/meta-yaml.html#python-entry-points

zip file for both platforms is fine, IMHO.  Not a huge issue, though.